pom.xml

```diff
@@ -16,6 +16,8 @@
         <maven.checkstyle.plugin.configLocation>
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
+        <maven.compiler.source>21</maven.compiler.source>
```

Comment:
The CI pipeline that executes the test for your code is using JDK17 - please keep your code in the same version so avoid compatibility issues.